### PR TITLE
Migration release

### PR DESCRIPTION
**Pull request by haojinming**
Add release pipeline for tikv/migration repo.
There are three products in tikv/migration repo
1. TiKV-BR
2. TiKV-CDC
3. TiKV-GC-Worker

There is one input parameter: `RELEASE_TAG`
The tag is defined as `productname-version` like: `br-v1.0.0` `cdc-v1.0.0` `gc-worker-v1.1.0`.

Jenkins pipeline will download binary zip files from github release and publish them to tiup with pingcap private key.

### CODE REVIEW

File: jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/*
+* @ RELEASE_TAG br-v1.0.0/cdc-v1.0.0/gc-worker-v1.0.0
```
Comment (zeminzhou) [inline jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy:2]: br-v1.0.0?

Comment (haojinming) [reply jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy:2]: This is just an format example.

File: jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    name=tikv-\${RELEASE_TAG%-v*}
+    version=v\${RELEASE_TAG#*-v}
+    desc=""
+    if [ \${name} == "tikv-br" ]; then
+        desc="TiKV cluster backup restore tool"
+    elif [ \${name} == "tikv-cdc" ]; then
+        desc="TiKV-CDC is a change data capture tool for TiKV"
+    elif [ \${name} == "tikv-gc-worker" ]; then
+        desc="GC Worker is a component for TiKV to control the gc process"
+    fi
```
Comment (pingyu) [inline jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy:35]: Suggest to raise an error or panic in `else` branch.

**Review by pingyu — COMMENTED** [review]
Rest LGTM.

**Review by zeminzhou — APPROVED** [review]
LGTM~

File: jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy (hunk 2 above)
Comment (haojinming) [reply jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy:38]: Done

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**